core/pom.xml

```diff
@@ -89,7 +84,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
```

Comment:
Bear in mind that core only uses Guava internally--we are not exporting that dependency by including Guava types in our API signatures.